docs/testing.rst

```diff
@@ -111,11 +111,11 @@ app test client::
             body='value: {}'.format(request.app[value]).encode('utf-8'))
 
     @pytest.fixture
-    def cli(loop, aiohttp_client):
+    def cli(event_loop, aiohttp_client):
         app = web.Application()
         app.router.add_get('/', previous)
         app.router.add_post('/', previous)
-        return loop.run_until_complete(aiohttp_client(app))
+        return event_loop.run_until_complete(aiohttp_client(app))
```

Comment:
```suggestion
       return await aiohttp_client(app)
```

Think it's probably better not to use the loop fixture at all 99% of the time these days.

Github is being too awkward for me to suggest the change in one comment...